### ISSUE TITLE
Possibility to sync recurring events with a calendar

### ISSUE THREAD

**Issue by kenza54** [firefly-iii/firefly-iii#3157]
Heya,

I'm wondering if it is possible to somehow sync the recurring events (or payments in general) with a calendar?

My usual workflow is calendar -> manual writing -> FF3 (I used to note down everything on calendar..) but I can't figure out if that is even possible.

Thanks!

**Comment by JC5** [comment]
I'm sorrry, but it's not. What is your workflow exactly? "Syncing" could mean a billion things.

**Comment by kenza54** [comment]
Since there was no iOS app back in the days I used to note down all my expenses (cash expenses which are 90% of my expenses anyway) in my Calendar. I am still doing that nowadays and then when I've got time I move them to FF3 (writing them manually that is).

The recurring events I have to add them on both FF3 + Calendar in order for me to remember I've got something to pay.

I'm just wondering if I can merge/sync the Calendar with the FF3 so I can stop writing twice everything.
